### PR TITLE
Added footer & changelog, new font

### PR DESCRIPTION
**Pull request by etsde**
- new fonts
- added changelog
- added footer with links
- general issue fixes
- dark mode fixes

### CODE REVIEW

File: css/style.css [hunk 1]
```diff
@@ -4,6 +4,54 @@
...
+    font-family: 'stolzl-book';
+}
+
+/*  Would not use differnet font styles, book only in different sizes is enough in my opinion. 
+    The following just does not look good.
+
+.weekday {
+    font-family: 'stolzl-bold';
+}
+*/
```
Comment (mr-woodapple) [inline css/style.css:50]: Can this be removed then?

File: css/style.css [hunk 2]
```diff
@@ -4,6 +4,54 @@
...
+
+
+
+
 @media (prefers-color-scheme: dark) {
 
     * {
```
Comment (mr-woodapple) [inline css/style.css:57]: What's the purpose of this? Can this be removed?

File: index.html [hunk 3]
```diff
@@ -3,28 +3,53 @@
...
+        }
+
+        footer a, a:hover, a:focus, a:active {
+            text-decoration: none;
+            color: inherit;
+            /* also all:unset; possible but not sure if it works on old browsers */
+        }
+
+        
+    </style>
```
Comment (mr-woodapple) [inline index.html:52]: Please move that to the .css file, messy to have .css in both the index.html and a style.css.

File: index.html [hunk 4]
```diff
@@ -39,15 +64,15 @@
...
 </head>
 <body>
-    
 
     <!-- Nav -->
     <nav class="navbar bg-body-secondary py-3">
         <div class="container-fluid">
             <div class="col-auto d-flex ">
-                <img src="/smartplan-logo.svg" width="15px" alt="">
+                <img src="img/smartplan-logo.svg" width="15px" alt="">
```
Comment (mr-woodapple) [inline index.html:75]: Really like that there's now an /img folder! 👍

File: index.html [hunk 5]
```diff
@@ -90,7 +116,20 @@ <h2 class="weekday"> Welcome! </h2>
...
     </div>
+
+    <!-- Footer -->
+    <footer class="text-center py-3 mt-auto">
+        <div class="container">
+            <hr class="vertical-line">
+            <p>&copy; 2023 bszet.me<br> All rights reserved.<br>
+                <a href='/CHANGELOG.md' class="footer">Changelog</a>
+                <span>|</span>
+                <a href='https://youtu.be/xvFZjo5PgG0?si=dLyMSXAD7srWbOw8' class="footer">GitHub</a>
```
Comment (mr-woodapple) [inline index.html:127]: You got me... 🙃

Comment (mr-woodapple) [reply index.html:127]: But for real, please change it ^^

File: index.html [hunk 6]
```diff
@@ -90,7 +116,20 @@ <h2 class="weekday"> Welcome! </h2>
         </div>
 
     </div>
+
+    <!-- Footer -->
+    <footer class="text-center py-3 mt-auto">
+        <div class="container">
+            <hr class="vertical-line">
+            <p>&copy; 2023 bszet.me<br> All rights reserved.<br>
```
Comment (mr-woodapple) [inline index.html:124]: Btw, I suggest removing that line, there are literally no rights or whatsoever we could reserve for ourselves.

File: index.html [hunk 7]
```diff
@@ -90,7 +116,20 @@ <h2 class="weekday"> Welcome! </h2>
...
+                <a href='/CHANGELOG.md' class="footer">Changelog</a>
+                <span>|</span>
+                <a href='https://youtu.be/xvFZjo5PgG0?si=dLyMSXAD7srWbOw8' class="footer">GitHub</a>
+            </p>
+        </div>
+    </footer>
```
Comment (mr-woodapple) [inline index.html:130]: Can we tone down the color of the text down there, maybe to this (as this is the color of the <hr /> tag): #5c5d5e?

File: css/style.css (hunk 1 above)
Comment (etsde) [reply css/style.css:50]: wanted to await @h3yFlo's thoughts

File: index.html (hunk 5 above)
Comment (etsde) [reply index.html:127]: where's the fun then? :D

File: index.html (hunk 7 above)
Comment (etsde) [reply index.html:130]: definitely!

File: index.html (hunk 3 above)
Comment (etsde) [reply index.html:52]: you're right, no idea why it's in here, tx

Comment (mr-woodapple) [reply index.html:52]: Update it please, then mark as resolved.

File: index.html (hunk 6 above)
Comment (mr-woodapple) [reply index.html:124]: Nah, I don't think that's a problem. And even if "all right reserved" wouldn't do anything to that, would need to be something like "all information are supplied without liability and all obligation".

File: index.html (hunk 5 above)
Comment (mr-woodapple) [reply index.html:127]: Gone, we're dead serious... 🥲

File: css/style.css (hunk 1 above)
Comment (mr-woodapple) [reply css/style.css:50]: I would just go with it, I like it (so remove this).

File: index.html (hunk 6 above)
Comment (etsde) [reply index.html:124]: then lets take this one

**Review by etsde — COMMENTED** [review]
- moved style from html to css
- edited footer
- removed unused lines

**Pull request merged**